libvirt/domain.go

```diff
@@ -252,6 +385,15 @@ func setCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain) err
 	return nil
 }
 
+func updateCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain, virConn *libvirt.Connect) error {
+	if runtime.GOARCH == "s390x" {
+		// There is no support for fw_cfg in s390x, so instead doing fw_cfg
```

Comment:
grammar: instead -> instead of.

Reply:
question: what about `s390` should that also be covered here?

Reply:
we never tested s390, but you are right, technically it should be ok to do so